kubernetes/README.md

```diff
@@ -1,32 +1,62 @@
-# Kubernetes manifest files
+# Kubernetes Manifest Files
 
 Under this directory, you can find the Kubernetes manifest files to deploy Falco and its components.
 
 ## Falco
 
 The [./falco](./falco) directory gives you the required YAML files to stand up Falco on Kubernetes as a DaemonSet. This will result in a Falco Pod being deployed to each node, and thus the ability to monitor any running containers for abnormal behavior.
 
-### Deploying to Kubernetes
+### Prerequisites and Template Limitations
+
+The default configuration in Falco utilizes the kernel module driver (`kmod`). The template daemonset setup in this repository is configured to deploy to the `default` namespace. To modify [falco.yaml](https://github.com/falcosecurity/falco/blob/master/falco.yaml) settings or experiment with different kernel drivers, refer to the [Install and Operate Guides](https://falco.org/docs/install-operate/), specifically the [Kubernetes Deployment](https://falco.org/docs/install-operate/deployment/) page. These guides provide insights on adjusting Falco's settings to suit various drivers, along with prerequisites for each. Also, check out the [libs](https://github.com/falcosecurity/libs) repository that contains the sources for each driver.
+
+For additional guidance on using Falco in Kubernetes or benchmarking within a testbed, consider exploring the repository at https://github.com/falcosecurity/cncf-green-review-testing. Please note that the repository primarily focuses on testbed benchmarking, and as a result, the setup might not be best for real-world use.
+
+__NOTE__: If you're working with [minikube](https://minikube.sigs.k8s.io/docs/start/) locally, bear in mind that specific mount setups might be necessary. For instance, running `minikube start --mount --mount-string="/usr/src:/usr/src" --driver=docker` ensures that the `driver-loader` can access `/usr/src/kernels/` if the kernel driver (`kmod` or `ebpf`) is built on-the-fly. Running Falco with `--modern-bpf` does not require building a driver because it is already bundled within the userspace binary. This capability is enabled by the newer CORE (Compile Once - Run Everywhere) BPF feature and works only on more recent kernels (>= 5.8).
+
+### What's Included in the `falco` Deployment?
+
+**initContainers**
+- `falco-driver-loader`: Downloads the kernel driver or attempts to build it on-the-fly.
+- `falcoctl-artifact-install`: Downloads default rules and installs falcoctl along with other artifacts like plugins.
+
+
+**containers**
+- `falco`: Executes the Falco binary.
+- `falcoctl-artifact-follow`: Utilizes falcoctl's functionality to watch for updated rules.
+
+The template daemonset setup does not handle the method of extracting Falco logs from the container to their final destination (such as a data lake or SIEM). You can explore using tools like `falco-exporter`, `falcosidekick`, or create custom solutions.
```

Comment:
Note for the future: this may slightly change a bit in Falco 0.37 since the porting of the driver loader into falcoctl cc @alacuku @Andreagit97 @LucaGuerra